File: CAPITULO 01/C++/ex19.c
#include <stdio.h>
int main(){
    int x,y,xx,yy;
    printf("Digite da altura do degrau (em centimetros):");
    scanf("%d%*c",&x);
    printf("Qual e a altura que deseja alcançar (em metros)? :");
    scanf("%d%*c",&y);
    yy = y * 1000;
    xx = yy /x;
    printf(" %.3d\n",xx);
    system("pause");
    return 0;

}

File: CAPITULO 01/Java/ex19.java
import java.util.*;
import java.io.*;
public class ex19{
	public static void main(String[]args){
		int x,y,z;
		float k;
		Scanner dado = new Scanner(System.in);
		System.out.println("Digite a altura do degrau (em centimetros):");
		x = dado.nextInt();
		System.out.println("Digite a altura que deseja alcançar (em metros): ");
		y = dado.nextInt();
		z = y * 1000;
		k =  z/x;
		System.out.println(" "+k);
	}
}